internal/pkg/admission/validating-webhook.go

```diff
@@ -19,13 +21,43 @@ func (r *Reconciler) reconcileValidatingWebhookConfiguration(
 	clusterAdmissionPolicy *policiesv1alpha2.ClusterAdmissionPolicy,
 	admissionSecret *corev1.Secret,
 	policyServerName string) error {
-	err := r.Client.Create(ctx, r.validatingWebhookConfiguration(clusterAdmissionPolicy, admissionSecret, policyServerName))
-	if err == nil || apierrors.IsAlreadyExists(err) {
+	webhook := r.validatingWebhookConfiguration(clusterAdmissionPolicy, admissionSecret, policyServerName)
+	err := r.Client.Create(ctx, webhook)
+	if err == nil {
 		return nil
 	}
+	if apierrors.IsAlreadyExists(err) {
+		return r.updateValidatingWebhook(ctx, clusterAdmissionPolicy, webhook)
```

Comment:
I wish this could be possible with a `createOrUpdate` kind of request instead of having to issue two requests, but seems the API does not allow this...

Reply:
yes, that would be great..